app/helpers/rules_of_origin_helper.rb

```diff
@@ -0,0 +1,21 @@
+module RulesOfOriginHelper
+  def rules_of_origin_service_name
+    TradeTariffFrontend::ServiceChooser.uk? ? 'UK' : 'EU'
+  end
+
+  def rules_of_origin_schemes_intro(country_name, schemes)
+    if schemes.empty?
+      return render 'rules_of_origin/intro_no_scheme', country_name: country_name
```

Comment:
I think the `( )` can make it a bit more tidy
return render('rules_of_origin/intro_no_scheme', country_name: country_name`)

up to you anyway, we do not have strictly rule for that 😄

Reply:
done